CHANGELOG.md

```diff
@@ -52,6 +52,7 @@ Keywords when running CAMSTATS.  [#3605](https://github.com/USGS-Astrogeology/IS
 - Added additional translation files for TGO CaSSiS in order to support PSA compliant labels. [#4567](https://github.com/USGS-Astrogeology/ISIS3/issues/4567)
 - Added support for KaguyaTC SP Support data ingest. [#4668](https://github.com/USGS-Astrogeology/ISIS3/issues/4668)
 - Added examples to the jigsaw documentation. [#4718](https://github.com/USGS-Astrogeology/ISIS3/issues/4718)
+- Added three examples to the jigsaw documentation covering relative and ground control bundles. [#4718](https://github.com/USGS-Astrogeology/ISIS3/issues/4718)
```

Comment:
I think we can just go with Stuart's original change log entry for all of these example updates. It's odd to have two changelog entries pointing to the same issue.

Reply:
I removed my entry, Stuart verbiage encompasses the addition of these examples